lab_3_generate_by_ngrams/main.py

```diff
@@ -105,6 +121,8 @@ def _put(self, element: str) -> None:
         an element is not added to storage
         """
 
+        if self._storage.get(i) == None:
```

Comment:
`None` is a singleton, better to check with `is`

Reply:
rewrited, thank you!!!